docs/Basic-Guides.md

```diff
@@ -0,0 +1,142 @@
+environment:
+
+1. Launch Unity.
+2. On the Projects dialog, choose the **Open** option at the top of the window.
+3. Using the file dialog that opens, locate the `unity-environment` folder 
+within the ML-Agents project and click **Open**.
+4. In the `Project` window, navigate to the folder 
+`Assets/ML-Agents/Examples/3DBall/`.
+5. Double-click the `3DBall` file to load the scene containing the Balance 
+Ball environment.
+
+![3DBall Scene](images/mlagents-Open3DBall.png)
+
+Since we are going to build this environment to conduct training, we need to 
+set the brain used by the agents to **External**. This allows the agents to 
+communicate with the external training process when making their decisions.
+
+1. In the **Scene** window, click the triangle icon next to the Ball3DAcademy 
+object.
+2. Select its child object `Ball3DBrain`.
+3. In the Inspector window, set **Brain Type** to `External`.
+
+![Set Brain to External](images/mlagents-SetExternalBrain.png)
+
+Next, we want the set up scene to play correctly when the training process 
+launches our environment executable. This means:
+* The environment application runs in the background
+* No dialogs require interaction
+* The correct scene loads automatically
+ 
+1. Open Player Settings (menu: **Edit** > **Project Settings** > **Player**).
+2. Under **Resolution and Presentation**:
+    - Ensure that **Run in Background** is Checked.
+    - Ensure that **Display Resolution Dialog** is set to Disabled.
+3. Open the Build Settings window (menu:**File** > **Build Settings**).
+4. Choose your target platform.
+    - (optional) Select “Development Build” to
+    [log debug messages](https://docs.unity3d.com/Manual/LogFiles.html).
+5. If any scenes are shown in the **Scenes in Build** list, make sure that 
+the 3DBall Scene is the only one checked. (If the list is empty, than only the 
+current scene is included in the build).
+6. Click *Build*:
+    a. In the File dialog, navigate to the `python` folder in your ML-Agents 
+    directory.
+    b. Assign a file name and click **Save**.
+
+![Build Window](images/mlagents-BuildWindow.png)
+
+Now that we have a Unity executable containing the simulation environment, we 
+can perform the training. You can ensure that your environment and the Python 
+API work as expected, by using the `python/Basics` 
+[Jupyter notebook](Background-Jupyter.md) introduced in the next section.
+
+## Using the Basics Jupyter Notebook
+
+The `python/Basics` [Jupyter notebook](Background-Jupyter.md) contains a 
+simple walkthrough of the functionality of the Python 
+API. It can also serve as a simple test that your environment is configured
+correctly. Within `Basics`, be sure to set `env_name` to the name of the 
+Unity executable you built earlier.
+
+More information and documentation is provided in the 
+[Python API](Python-API.md) page.
+
+## Training the Brain with Reinforcement Learning
+
+Go to your command line, enter the `ml-agents/python` directory and type: 
```

Comment:
I've found that many developers need a surprising amount of handholding when it comes to running programs on the command line. For example, many people will type in the following line verbatim and wonder why it doesn't work. I would simplify and use a concrete example so the only thing a developer has to change is the env_name.

Reply:
I agree with this, but how do we describe "enter the ml-agents/python" directory here? 
Do we say
```
cd <The Directory of ML-Agents>/python
python3 learn.py <env_name> --run-id=<run-identifier> --train 
```
?

Reply:
I generally just describe it:

1. Open a command or terminal window.
2. Navigate to the folder where you installed ML-Agents.
3. Change to the `python` directory:

    `cd python`

4. Run `python3 learn.py <env_name> --run-id=<run-identifier> --train`
    Where: 
        - `<env_name>` is the name and path to the executable you exported from Unity (without extension)
        - `<run-identifier>` is a string used to separate the results of different training runs
        - And the `--train` tells learn.py to run a training session (rather than inference)

    For example, if you are training with a 3DBall executable you exported to the `ml-agents/python` directory, run:

    `python3 learn.py 3DBall --run-id=firstRun --train`

-------

An alternate approach is to lead with the example, which has the benefit of bing a bit shorter, though it doesn't explain as much:

1. Open a command or terminal window.
2. Navigate to the folder where you installed ML-Agents.
3. Change to the `python` directory:

    `cd python`

4. Run:

     `python3 learn.py 3DBall --run-id=firstRun --train`

    (If you are training using a different environment, substitute `3DBall` with the name and path to the training executable you exported from Unity.)

Reply:
I really like what you suggested, I will change it to your first version.